### PR TITLE
Hangman

### PR DESCRIPTION
**Pull request by dovydasjaru**

### CODE REVIEW

File: WhosThatAndroid/WhosThatAndroid/Activities/HangmanActivity.cs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    class HangmanActivity : Activity
+    {
+        private ImageView DisplayImg { get; set; }
+        private LinearLayout DisplayWord { get; set; }
+        private Button MyButton { get; set; }
+        private EditText Input { get; set; }
+        private Button BtnNewGame { get; set; }
+        private List<string> WordList { get; set; }
+        private string Word { get; set; }
+        private Animator MyAnimator { get; set; }
```
Comment (Aurintas) [inline WhosThatAndroid/WhosThatAndroid/Activities/HangmanActivity.cs]: Maybe name this file and variable in a meaningful way? For instance, HangmanAnimator.

File: WhosThatAndroid/WhosThatAndroid/Activities/MeniuActivity.cs [hunk 2]
```diff
@@ -10,30 +10,38 @@
...
     class MeniuActivity : Activity
     {
         protected override void OnCreate(Bundle savedInstanceState)
         {
             base.OnCreate(savedInstanceState);
             SetContentView(Resource.Layout.MeniuWindow);
 
-            var liveRecCard = FindViewById(Resource.Id.liveRecCard);
-            liveRecCard.Click += openActivity(typeof(LiveRecognitionActivity));
+            var card = FindViewById(Resource.Id.liveRecCard);
```
Comment (LinasEin) [inline WhosThatAndroid/WhosThatAndroid/Activities/MeniuActivity.cs]: I believe, if you do like this, whatever you press you'll get to PhotoRecActivity. As Eventhandler for that click event was initialized the last. Have you tested?

Comment (dovydasjaru) [reply WhosThatAndroid/WhosThatAndroid/Activities/MeniuActivity.cs]: I tested and it works

File: WhosThatAndroid/WhosThatAndroid/Activities/HangmanActivity.cs [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        {
+            base.OnCreate(bundle);
+            SetContentView(Resource.Layout.HangmanWindow);
+
+            DisplayImg = FindViewById<ImageView>(Resource.Id.hangmanImage);
+            DisplayWord = FindViewById<LinearLayout>(Resource.Id.hangmanDisplayWord);
+            BtnNewGame = FindViewById<Button>(Resource.Id.buttonNewGame);
+            BtnNewGame.Click += BtnNewGameClick;
+            WordList = new List<string>();
+            WordList.Add("word");
```
[the remark this answers is not in the document]
Comment (dovydasjaru) [reply WhosThatAndroid/WhosThatAndroid/Activities/HangmanActivity.cs]: I plan to get word from database. So up until then this is just for testing

**Comment by Aurintas** [comment]
Assets editor should same file but with different sizes for each directory (drawable-xdpi, drawable-mdpi..) and without suffix .9, which is used for different purpose (when drawable is meant to be repeated multiple times).

**Pull request merged**